### DISCUSSION TITLE
[Implementation inside] Add ability for sidecar to be sorted on reference page

### DISCUSSION THREAD

**Discussion by cristeahub** [ghostty-org/ghostty#3397] [Feature Requests, Ideas]
After looking at the issues on the webpage repo, I opted to create a PR directly https://github.com/ghostty-org/website/pull/202

---

As I am currently configuring my Ghostty, I often times see that I want to look up various configs and see what values are available for these configs.

When browsing the reference (https://ghostty.org/docs/config/reference) I would love for the sidecar to be sorted, so it's easy to find the values I'm looking for.

I have implemented this as an optional value per page here: https://github.com/ghostty-org/website/compare/main...cristeahub:ghostty-website:cto/add-sorted-sidecar

If this is something that is wanted I will open a pull request.

Before (sidecar unsorted):
<img width="1643" alt="image" src="https://github.com/user-attachments/assets/df9bd5b3-a2ed-45b0-ab3f-1952e206d2ca" />
After (sidecar sorted):
<img width="1643" alt="image" src="https://github.com/user-attachments/assets/4c11af88-4870-4c77-9461-9458ea1e7ddb" />

**Comment by BrandonRomano** [comment]
The sidecar sorting should remain the same as the content itself, this is a pretty well established web/docs pattern that I don't want to break from.

There's definitely room for improvement though on the scanability of this page, though & I am open to other ideas.

**Comment by cristeahub** [comment]
A solution would be to sort the actual content itself, however that will in my opinion cause the pages to worsen slightly, since one would most likely show the most common config first.

I guess one solution would be to have the page be fully sortable, i.e. letting the content either be in curated order, or in sorted order.
